### ISSUE TITLE
Ability to limit the library scan by date

### ISSUE THREAD

**Issue by siburny** [mtalcott/google-photos-deduper#19]
Currently, it tries to download ALL photos from my Google Photos library. But having more than 100K images it's easy to go over the API quota. There should be either an option to specify how far back I want the app to scan my library (e.g. last 3 years) OR two date fields to specify start and end date for the scan.

**Comment by mtalcott** [comment]
Good idea. I'd consider that one of the filtering options that would be helfpul alongside some others mentioned over on https://github.com/mtalcott/google-photos-deduper/issues/7.

Unfortunately, the Google Photos API is fairly limited and does not include a field for when media items were added to the library. It does [include a `creationTime` within the metadata](https://developers.google.com/photos/library/reference/rest/v1/mediaItems#MediaItem), but that comes from the image metadata and uploaded. So the filtering by date wouldn't really work as people would expect.
